create.sql

```diff
@@ -266,6 +266,7 @@ CREATE TABLE IF NOT EXISTS `users` (
   `login_expiry` bigint(20) COLLATE utf8_unicode_ci DEFAULT '-1',
   `last_login` bigint(20) COLLATE utf8_unicode_ci DEFAULT '-1',
   `login_timeout` smallint(5) COLLATE utf8_unicode_ci DEFAULT '-1',
+  `id` INT NOT NULL AUTO_INCREMENT UNIQUE KEY,
```

Comment:
Let's use a BIGINT as column type and move it as first col, so we can later use ID as primary key instead of username